AutoNode/common.py

```diff
@@ -0,0 +1,60 @@
+import json
+import subprocess
+import os
+
+daemon_name = "autonoded"
+harmony_dir = f"{os.environ['HOME']}/.hmy"
+node_dir = f"{os.environ['HOME']}/harmony_node"
+node_sh_log_dir = f"{node_dir}/node_sh_logs"
+bls_key_dir = f"{harmony_dir}/blskeys"
+imported_bls_pass_file_dir = f"{os.environ['HOME']}/bls_pass"
+imported_wallet_pass_file_dir = f"{os.environ['HOME']}/wallet_pass"
+cli_bin_dir = f"{harmony_dir}/bin"
+cli_bin_path = f"{cli_bin_dir}/hmy"
+saved_validator_path = f"{os.environ['HOME']}/validator_config.json"
+saved_node_path = f"{harmony_dir}/.saved_config"
+saved_wallet_pass_path = f"{harmony_dir}/.wallet_pass"
+node_pid_path = f"{harmony_dir}/.autonode_node_pid"
+
+node_script_source = "https://raw.githubusercontent.com/harmony-one/harmony/master/scripts/node.sh"
```

Comment:
how we know we're using the right node.sh? hash the file content, something like that

Reply:
That would be a good idea. For the time being, I'm going to trust the https. Adding check is something to do in the future.